index.html

```diff
@@ -9,9 +9,12 @@
   </head>
   <body>
     <h1>Splatoon</h1>
+    <h2>Score <span></span></h2>
     <div id="canvas"></div>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.8.0/p5.min.js"></script>
     <script src="src/constants.js"></script>
+    <script src="src/player.js"></script>
+    <script src="src/Treasure.js"></script>
     <script src="src/game.js"></script>
```

Comment:
For consistency sakes, you could also have capitalized Player.js and Game.js since they're both classes as well! ✌